src/io-methods/s3.js

```diff
@@ -21,6 +21,17 @@ function parseFilePath(filePath) {
   };
 }
 
+const validCannedACLs = [
+  // See: https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl
+  "private", "public-read", "public-read-write", "aws-exec-read", "authenticated-read",
+  "bucket-owner-read", "bucket-owner-full-control", "log-delivery-write"
```

Comment:
according to https://developers.digitalocean.com/documentation/spaces/ only `private` and `public-read` are available for DO

what should we do in that case? should we be checking to see if they're using DO and then validate against that set of options?

or should we remove the validation altogether so that we don't have to ensure we're always up-to-date with 2 external services?

thoughts?

Reply:
Removed validation outright, since we're going to just pass along the s3 putObject config.